calcom/cal.com
# removed SSR trpc calls instead SPA for logged in pages

fixes https://github.com/calcom/cal.com/issues/6935


so far so good on localhost

## Why SSR in the first place: 

we used to have i18n flickering with CSR but i cant reproduce them anymore. ideally we dont rely on SSR for logged in pages, since its slower

## what to test

- hard reload some pages
- see if i18n keys are flickering
- change language in settings
